### PR TITLE
AppReviewsDialog: improve padding

### PR DESCRIPTION
**Pull request by Feichtmeier**
|Before|After|
|-|-|
|image|image|

### CODE REVIEW

File: lib/app/common/app_page/app_reviews.dart [hunk 1]
```diff
@@ -162,32 +162,40 @@ class _ReviewDetailsDialog extends StatelessWidget {
           ? []
           : userReviews!
               .map(
-                (e) => BorderContainer(
+                (e) => Padding(
```
Comment (jpnurmi) [inline lib/app/common/app_page/app_reviews.dart:165]: you can pass `margin` to avoid having to wrap Container or YaruBorderContainer with Padding

Comment (Feichtmeier) [reply lib/app/common/app_page/app_reviews.dart:165]: no, this is the old discussion :D padding in the container adds padding around the child :)
here we need both, around the child and around the container

Comment (Feichtmeier) [reply lib/app/common/app_page/app_reviews.dart:165]: image

Comment (Feichtmeier) [reply lib/app/common/app_page/app_reviews.dart:165]: eeeeeeeh margin :face_exhaling: ok sec

Comment (Feichtmeier) [reply lib/app/common/app_page/app_reviews.dart]: done :facepalm:

**Review by jpnurmi — APPROVED** [review]
LGTM :+1:

**Pull request merged**